apps/emqx_bridge_http/rebar.config

```diff
@@ -0,0 +1,10 @@
+%% -*- mode: erlang; -*-
+{erl_opts, [debug_info]}.
+{deps, [ {emqx_connector, {path, "../../apps/emqx_connector"}}
+       , {emqx_resource, {path, "../../apps/emqx_resource"}}
+       , {emqx_bridge, {path, "../../apps/emqx_bridge"}}
+       ]}.
```

Comment:
Strange that this works without declaring `ehttpc` as a dependency.  :thinking:

Reply:
we have it as a dependency on the top level